src/services/jobs.js

```diff
@@ -0,0 +1,15 @@
+import { get } from 'Services/http';
+
+const API_URL = process.env.COMPANIES_API_URL;
+
+export const getJobs = async () => {
+    const { data } = await get(`${API_URL}vacancies`);
```

Comment:
this endpoint brings the entire list of vacancies and you don't have to change the array https://gethiredplatzi.herokuapp.com/api/v1/vacancies

we also need to add a filter to filter the active offers